tutorials/0. Theory.ipynb

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
<p>Add comment explaining what is <code>PyDEns</code> ("framework for solving Partial Differentials Equations (PDEs) using neural networks..."), to make it clear why the math behind it is interesting in the first place.</p><p> </p><p>I'd also write <code>parametric families of PDEs</code> instead of  <code>parametric equations</code> for further clarification - afterall, this is an introductory paragraph.</p><p> </p><p>..it is structured ad follows: typo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/pydens/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>